### DISCUSSION TITLE
Is there a timeline to expect for WinRT activation blocker  fix?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/WindowsAppSDK#813] [Q&A] [answered]
as per the answer of this question [My blank app does not work properly with elevated privilege ?](https://github.com/microsoft/ProjectReunion/discussions/671#discussioncomment-585727) 

> Unfortunately, WinUI 3 currently does not support running elevated. This is due to a limitation in WinRT in the OS (WinRT activation doesn't work in elevated apps), and we'll hopefully be fixing that in a future version of Windows.
 
_WinRT activation in elevated apps_  issue in the OS  said to be fixed in future versions of windows.  
as per roadmap Reunion 0.8 will be released in Q2
Can we expect this WinRT fix when Reunion 1.0 gets released ? 
sharing even a vague timeline will do.
thanks.

**Comment by contextfree** [comment, marked as answer]
It sounds like they're saying the fix won't be part of the Project Reunion add-on package but will have to be part of a Windows feature update (21H2 or later?)

**Reply by ghost** [reply]
maybe. need explicit clarification from MS.

**Comment by ghost** [comment]
@andrewleader as future reunion APIs are going to be  WinRT apis, getting it to work on high IL is crucial one.

can you please say when the Fix is coming ?    ignoring native api functionalities sends bad vibes for sure.

**Comment by andrewleader** [comment]
Hi @ecovio1, thanks for bringing this up, I was thinking contextfree's answer seemed to answer the question which is why I didn't answer originally, but I apologize for the delay.

This is a limitation in the OS itself that needs to be fixed in the OS itself. Right now, all we can disclose is that we'll hopefully be fixing this in a future version of Windows.
